### ISSUE TITLE
Close event callback

### ISSUE THREAD

**Issue by howdu** [Simonwep/pickr#49]
Is there a callback for closing? 

I want to show a preview to the user on change, cancel removes the color and closing the picker revert back to default. This is what I currently have:
```javascript
const el = document.querySelector('.picker-open');
const shape = document.querySelector('.picker-shape');

function updateShapeColor(color) {
  if (color) {
    shape.style.backgroundColor = color;
  } else {
    shape.style.backgroundColor = 'transparent';
  }
}

var lastPickedColor;

const pickr = new Pickr({
  el,
  default: 'rgb(215, 223, 33)',
  defaultRepresentation: 'RGBA',
  onChange: function(hsva) {
    if (hsva) {
      updateShapeColor(hsva.toRGBA().toString());
    }
  },
  onSave: function(hsva) {
    if (hsva) {
      lastPickedColor = hsva.toRGBA().toString();
      updateShapeColor(lastPickedColor);
    } else {
      updateShapeColor('');
    }
  },
  onClose: function() {
  	if (lastPickedColor) updateShapeColor(lastPickedColor);
  },
  components: {
    preview: true,
    opacity: true,
    hue: true,
    comparison: false,
    interaction: {
      hex: true,
      rgba: true,
      hsla: false,
      hsva: false,
      cmyk: false,
      input: true,
      clear: true,
      save: true
    }
  },
  strings: {
    save: 'Select',
    clear: 'Cancel'
  }
});

```

https://jsfiddle.net/2ga3eckL/

**Comment by Simonwep** [comment]
Unfortunately not, but you can put this in your else branch in `onSave` as temporary solution:
```js
/**
 * _lastColor is, well, the last color. So get the HSVA array and
 * apply it. This triggeres a re-draw and moreover a onSave call.
 */
pickr.setHSVA(...pickr._lastColor.toHSVA());
```

See https://jsfiddle.net/ayq1xv69/4/
But good idea, I'll take such a function in consideration!

**Comment by Simonwep** [comment]
Okay, I decided to not implement such a callback. I'd also need to add a `onShow` etc. listener which is kind of overkill. Anyway there is still the possibility to _override_ `onShow` and `onHide` to get this kind of feature.

**Comment by Simonwep** [comment]
Currently there's a plan where such a thing will be available, see #66 :)
